lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -56,12 +56,12 @@
       </a>
         % else:
         <a class="fade-cover">
-        <img src="${course_image_url(course)}" alt="${_('{course_number} {course_name} Cover Image').format(course_number=course.number, course_name=course.display_name_with_default) |h}" />
+        <img src="${course_image_url(course)}" alt="${_('{course_number} {course_name} Home Page').format(course_number=course.number, course_name=course.display_name_with_default) |h}" />
```

Comment:
@clrux I guess we don't want empty `alt` here as this image links to the home page of the course so if we removed the alt tag here the screen reader users would no longer able to know the purpose of link image